README.md

```diff
@@ -275,6 +275,20 @@ event = DfE::Analytics::Event.new
   .with_data(some: 'custom details about event')
 ```
 
+If you need to include hidden_pii, you can use the `hidden_data` key which will allow all fields lsited to be sent separately to BigQuery where they will be hidden.
+
+```ruby
+event = DfE::Analytics::Event.new
+  .with_type(:some_custom_event)
+  .with_user(current_user)
+  .with_request_details(request)
+  .with_namespace('some_namespace')
+  .with_data(
+    data: { some: 'custom details about event' },
```

Comment:
Would it be worth including more than one field in data and hidden_data in the example so it's super clear how to achieve this?

Reply:
Now updated